sig-node/charter.md

```diff
@@ -0,0 +1,74 @@
+# SIG Node Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Node is responsible for the components that support the controlled
+interactions between pods and host resources.  We manage the lifecycle of pods
+that are scheduled to a node.  We focus on enabling a broad set of workload
+types, including hardware and performance sensitive workloads.  We maintain
```

Comment:
I have trouble parsing "including hardware and performance sensitive workloads".